ui/public/index.html

```diff
@@ -6,7 +6,7 @@
   <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
-  <meta name="description" content="Machine Learning Platform" />
+  <meta name="description" content="CaraML" />
```

Comment:
Updated this to make the migration consistent with the future changes to the MLP UI i.e. renaming of 'Machine Learning Platform' to 'CaraML'.